packages/theme_tailor_annotation/lib/src/theme_tailor_component.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:meta/meta_meta.dart';
+import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
+
+const tailorComponent = TailorComponent();
+
+@Target({TargetKind.classType})
```

Comment:
Also can add docs here, maybe smth like:
"An annotation used to specify that generated ThemeExtension class can be used as a part of other ThemeExtension class and does not need additional extensions on BuildContext nor ThemeData for the ease of access"